kyo-core/shared/src/test/scala/kyoTest/Tagged.scala

```diff
@@ -0,0 +1,8 @@
+package kyoTest
+
+import org.scalatest.Ignore
+import org.scalatest.Tag
+
+object Tagged:
+    val jvmOnly = Tag(if Platform.isJVM then "" else classOf[Ignore].getName)
```

Comment:
@fwbrasil I added this so that the tests will properly be marked as `ignored` in the console. Let me know what you think.

Reply:
Cool, I didn't know about `Tag`. LGMT. Is it possible to tag individual tests as well?

Reply:
Actually, due to the inclusion of `AsyncFreeSpec` that's the only reasonable way to use Tags, unless we define them as a java annotation - which I am not inclined to do.